posix.mak

```diff
@@ -130,14 +108,16 @@ $(CURL_TOOLS): $(ROOT)/%: %.d
 $(TOOLS) $(DOC_TOOLS): $(ROOT)/%: %.d
 	$(DMD) $(MODEL_FLAG) $(DFLAGS) -of$(@) $(<)
 
-$(TAGS): %.tag: $(DOC)/%.html $(ROOT)/findtags
-	$(ROOT)/findtags $< > $@
+$(DOC)/chmgen : $(DOC)/chmgen.d
+	$(DMD) -g -of$@ $<
+
+ALL_OF_PHOBOS_DRUNTIME_AND_DLANG_ORG = # ???
```

Comment:
> How about you pass the DOCS folder to chmgen and let it find all HTML files.

It already finds HTML files by itself (though the directory is currently hardcoded). The difficulty is Make target dependencies (i.e. we want dman.exe to be rebuilt iff lex.dd changed).